### ISSUE TITLE
There is no option of selecting `limit_multi_searches` in `multiSearch` using typescript.

### ISSUE THREAD

**Issue by MarkoRisticSting** [typesense/typesense-js#162]
## Description

There is no option of selecting `limit_multi_searches` in `multiSearch` using typescript.

## Steps to reproduce

typescript complains

<img width="585" alt="Screenshot 2023-06-20 at 20 49 44" src="https://github.com/typesense/typesense-js/assets/99866757/9e6019f0-37db-4120-85ca-2f9911f1265f">

## Expected Behavior

To already have option for `limit_multi_searches` in `MultiSearchRequestSchema`
We can see that it is missing.

<img width="520" alt="Screenshot 2023-06-20 at 20 52 52" src="https://github.com/typesense/typesense-js/assets/99866757/950c57fc-676b-4590-bbec-c891b0eaef34">

## Actual Behavior

TS2345: Argument of type `{ collection: string; limit_multi_searches: number; }' is not assignable to parameter of type 'Partial<MultiSearchRequestSchema>`.

## Metadata

1.5.4

Mac OS

**Comment by jasonbosco** [comment]
Ah, will fix the types for this. On a side note, even though the multi_search endpoint supports `limit_multi_searches`, I would recommend embedding it in a Scoped Search API key, so it cannot be modified.

**Comment by MarkoRisticSting** [comment]
Thanks for quick response.

**Comment by jasonbosco** [comment]
Yeah, I'll probably just add `limit_multi_searches` only to `GenerateScopedSearchKeyParams` to encourage recommended usage of this param.

**Comment by jasonbosco** [comment]
Could you try this with `v1.6.0-1`?

**Comment by MarkoRisticSting** [comment]
Typescript doesn't complain anymore, let me figure out how Scoped Search API key works now 😂 thanks again.
<img width="722" alt="Screenshot 2023-06-20 at 22 22 44" src="https://github.com/typesense/typesense-js/assets/99866757/df42e0ff-8530-4c14-aea4-38b25b582b0b">
